kafka-on-docker/README.md

```diff
@@ -0,0 +1,145 @@
+<!-- title: How to run Kafka locally on Docker -->
+<!-- description: In this tutorial, learn how to run Kafka broker locally on your laptop, with step-by-step instructions and supporting code. -->
+
+# How to run a Kafka broker locally with Docker
+
+When building or testing Apache®Kafka applications, it can be very helpful to have a local Kafka broker running locally.  Using [Docker](https://www.docker.com/) is possibly the fastest way to get a locally running Kafka broker
+
+## Set up
+
+To run Kafka on Docker, you'll need to install [Docker Desktop](https://www.docker.com/products/docker-desktop/). Then, after you've installed Docker Desktop, you'll use [Docker Compose](https://docs.docker.com/compose/) to create your Kafka container.  Docker Compose uses a YAML configuration file to manage your Docker components (services, volumes, networks, etc.) in an easy-to-maintain approach.  Docker Desktop includes Docker Compose, so there are no additional steps you need to take.
+Finally, you'll need an account on [docker hub](https://hub.docker.com/explore) so Docker Desktop can pull the images specified in the Docker Compose file.
```

Comment:
```suggestion
Finally, you'll need an account on [Docker Hub](https://hub.docker.com/explore) so that Docker Desktop can pull the images specified in the Docker Compose file.
```